gravitee-apim-console-webui/src/management/api/health-check-dashboard-v4/components/availability-per-field/availability-per-field.component.ts

```diff
@@ -117,6 +117,11 @@ export class AvailabilityPerFieldComponent implements OnInit {
 
   mergeGroups = (responseTimes: ApiAverageResponseTime, availability: ApiAvailability): TableData[] => {
     const tableData: TableData[] = [];
+
+    if (responseTimes == null || responseTimes.group == null || availability == null || availability.group == null) {
```

Comment:
😉 3 '='
can be write
```suggestion
    if (!responseTimes?.group || !availability?.group) {
```

Reply:
but we reducing readability